examples/ct/ct_handlers.go

```diff
@@ -44,23 +62,40 @@ type addChainRequest struct {
 	Chain []string
 }
 
+type addChainResponse struct {
+	SctVersion int    `json:sct_version`
+	ID         string `json:id`
+	Timestamp  string `json:timestamp`
+	Extensions string `json:extensions`
+	Signature  string `json:signature`
+}
+
 func parseBodyAsJSONChain(w http.ResponseWriter, r *http.Request) (addChainRequest, error) {
 	body, err := ioutil.ReadAll(r.Body)
 
 	if err != nil {
-		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		if glog.V(logVerboseLevel) {
```

Comment:
Only if the arguments are expensive to evaluate, which they aren't in any of these cases.